src/applications/caregivers/definitions/caregiverUI.js

```diff
@@ -36,7 +36,7 @@ export default {
   },
   sharedItems: {
     contactInfoTitle: 'Contact information',
-    fullNameUI: label => ({
+    fullNameUI: (label) => ({
```

Comment:
Doesn't adding these parentheses cause a linting error?

Reply:
Yeah I think something happened and default prettier or something was applied rather than our workspace rules